OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitWebViewFragment.java

```diff
@@ -226,6 +229,14 @@ public void updateDrawState(@NotNull TextPaint textPaint) {
 
         tvOpenBrowser.setText(openInBrowserSpan);
         tvOpenBrowser.setMovementMethod(LinkMovementMethod.getInstance());
+        trackOpenInBrowserBannerEvent(Analytics.Events.OPEN_IN_BROWSER_BANNER_DISPLAYED,
+                Analytics.Values.OPEN_IN_BROWSER_BANNER_DISPLAYED);
+    }
+
+    private void trackOpenInBrowserBannerEvent(String eventName, String biValue) {
+        environment.getAnalyticsRegistry().trackOpenInBrowserBannerEvent(eventName, biValue,
+                enrollmentMode, unit.getCourseId(), unit.getBlockId(),
```

Comment:
We need to send the component id instead of block id. So please can you update the following: 
`unit.getBlockId()` to `unit.getId()`